tasks/models.py

```diff
@@ -124,7 +124,7 @@ class Task(models.Model):
     )
 
     repeats = models.PositiveBigIntegerField(
-        _("Hoe vaak wordt deze taak uitgevoerd binnen deze sessie?"),
+        _("Hoe vaak wordt deze taak uitgevoerd binnen deze sessie (per deelnemer)?"),
```

Comment:
I think this is overly terse wording. How about: "Hoe vaak voert een deelnemer deze taak uit binnen deze sessie?"

The English translation could be "How often will a participant perform this task within this session?"